Core/src/org/sleuthkit/autopsy/centralrepository/eventlisteners/CaseEventListener.java

```diff
@@ -235,15 +234,15 @@ private void handleTagDeleted(ContentTagDeletedEvent evt) {
                 // then handle the event 
                 handleTagChange(content);
             } catch (NoCurrentCaseException | TskCoreException ex) {
-                LOGGER.log(Level.WARNING, "Error updating non-file object.");
+                Long contentID = (evt != null && evt.getDeletedTagInfo() != null) ? evt.getDeletedTagInfo().getContentID() : null;
```

Comment:
Is this check necessary? We already know evt.getDeletedTagInfo() is not null, and the call to getContentID() can not throw an exception.

Reply:
Sure.